src/main/java/com/fasterxml/jackson/core/util/TextBuffer.java

```diff
@@ -964,7 +960,7 @@ private void unshare(int needExtra)
     }
 
     // Method called when current segment is full, to allocate new segment.
-    private void expand(int minNewSegmentSize)
```

Comment:
this input value is always ignored - so I've stopped sending a value in the calls

Reply:
Hmmh. That is probably not a safe thing to do, theoretically. But I'd need to have a look at PR in its entirety.

Reply:
I've moved this change to its own PR - https://github.com/FasterXML/jackson-core/pull/877